README.md

```diff
@@ -140,6 +140,8 @@ docker create \
     --env SUBSPACE_IPV6_GW="fd00::10:97:1" \
 	# Optional variable to enable or disable IPv6 NAT
     --env SUBSPACE_IPV6_NAT_ENABLED=1 \
+  # Optional variable to disable iptables rules if you want to manage them yourself
+    --env SUBSPACE_IPTABLES_DISABLED=1 \
```

Comment:
This can be achieved by disabling IPV6 and IPV4 NAT Enabled, this adds a third variable that isn't required

Reply:
Ah I didn't see those updates over the old entrypoint.sh. Still leaves DNS Leak Protection though.